components/user_settings/general/user_settings_general.tsx

```diff
@@ -856,7 +893,7 @@ class UserSettingsGeneralTab extends React.PureComponent {
                 />
             );
         } else {
-            let describe = '';
+            let describe: any = '';
```

Comment:
This and all others like it can actually be of type `React.ReactNode`. It includes all of the possible types that can be rendered to the DOM. Sorry about the confusion.